spec/traits.dd

```diff
@@ -438,12 +438,12 @@ $(H2 $(GNAME getAliasThis))
 
 $(SECTION2 $(GNAME getAttributes),
     $(P
-        Takes one argument, a symbol. Returns a tuple of all attached user defined attributes.
+        Takes one argument, a symbol. Returns a tuple of all attached user-defined attributes.
         If no UDA's exist it will return an empty tuple.
     )
 
     $(P
-        For more information, see: $(DDSUBLINK spec/attribute, uda, User Defined Attributes)
+        For more information, see: $(DDSUBLINK spec/attribute, uda, User-defined Attributes)
```

Comment:
Probably should capitalize `defined` here, to be consistent with other places where it's capitalized?

Reply:
Yeah, it's in a title. I think standard English rules should apply.

Reply:
OK. changed.